purchases/src/test/java/com/revenuecat/purchases/PostingTransactionsTests.kt

```diff
@@ -103,7 +103,12 @@ class PostingTransactionsTests {
                 every { currentAppUserID } returns appUserId
             },
             subscriberAttributesManager = subscriberAttributesManagerMock,
-            appConfig = AppConfig("en-US", "1.0", PlatformInfo("native", "3.2.0"), true)
+            appConfig = AppConfig(
+                context = mockk(relaxed = true),
+                observerMode = false,
+                platformInfo = PlatformInfo(flavor = "native", version = "3.2.0"),
+                proxyURL = null
+            )
```

Comment:
This is a regression from merging https://github.com/RevenueCat/purchases-android/pull/148